java/README.md

```diff
@@ -49,6 +49,8 @@
   ```
   * Check that Docker is up and running
   * If you use an alternative to Docker, check [General Container runtime requirements for Testcontainers](https://java.testcontainers.org/supported_docker_environment/)
+  * If you are still having trouble launching test container, you can use [test containers cloud](https://testcontainers.com/cloud/). It's free for solo developers
```

Comment:
```suggestion
  * In case you are still having trouble launching Testcontainers, you can use [Testcontainers Cloud](https://testcontainers.com/cloud/). It is free for solo developers.
```